lint/rule_name.go

```diff
@@ -31,19 +31,12 @@ const nameSeparator string = "::"
 var ruleNameValidator = regexp.MustCompile("^([a-z0-9][a-z0-9-]*(::[a-z0-9][a-z0-9-]*)?)+$")
 
 // getRuleGroup takes an AIP number and returns the appropriate group.
+// It panics if no group is found.
 func getRuleGroup(aip int) string {
-	// Determine the group.
-	group := ""
 	if aip > 0 && aip < 1000 {
-		group = "core"
+		return "core"
 	}
-
-	// Sanity check: If the group does not exist, complain.
-	if group == "" {
-		panic("Invalid AIP; no available group.")
-	}
-
-	return group
+	panic(fmt.Sprintf("Invalid AIP number %d: no available group.", aip))
```

Comment:
Is this okay? I usually get lint warnings when the last statement of a function is not a `return` statement.

Reply:
Return or panic seems very reasonable to me. I run `golangci-lint` and `golint`, and didn't find any warning about this change.